docs/pages/reference/helm-reference/teleport-plugin-mattermost.mdx

```diff
@@ -0,0 +1,251 @@
+| `string` | `""` | Yes |
+
+Base URL of the MatterMost instance
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  mattermost:
+    url: "https://mattermost.example.com/"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set mattermost.url="https://mattermost.example.com/"
+  ```
+  </TabItem>
+</Tabs>
+
+## `mattermost.token`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | No |
+
+MatterMost token of the bot user to impersonate when sending the messages.
+Ignored when `mattermost.tokenFromSecret` is set.
+It's only recommended for testing purposes. Please use
+[`mattermost.tokenFromSecret`](#mattermosttokenfromsecret) instead.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  mattermost:
+    token: "xoxb-1234"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set mattermost.token="xoxb-1234"
+  ```
+  </TabItem>
+</Tabs>
+
+## `mattermost.tokenFromSecret`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | No |
+
+Secret containing the MatterMost token of the bot user.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  mattermost:
+    tokenFromSecret: "teleport-mattermost-plugin-token"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set mattermost.tokenFromSecret="teleport-mattermost-plugin-token"
+  ```
+  </TabItem>
+</Tabs>
+
+## `mattermost.tokenSecretPath`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `"mattermostToken"` | No |
+
+Key where the token is located inside the secret specified by `mattermost.tokenFromSecret`.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  mattermost:
+    tokenSecretPath: "token"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set mattermost.tokenSecretPath="token"
+  ```
+  </TabItem>
+</Tabs>
+
+## `mattermost.recipients`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `array` | `[]` | Yes |
+
+List of recipients who will receive notifications about access requests.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  mattermost:
+    recipients:
+      - user1@example.com
+      - user2@example.com
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set mattermost.recipients[0]="user1@example.com",mattermost.recipients[0]="user1@example.com"
+  ```
+  </TabItem>
+</Tabs>
+
+## `log.output`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `stdout` | No |
+
+Logger output. Could be "stdout", "stderr" or a file name, eg. "/var/log/teleport/mattermost.log"
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  log:
+    output: /var/log/teleport/mattermost.log
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set log.output="/var/log/teleport/mattermost.log"
+  ```
+  </TabItem>
+</Tabs>
+
+## `log.severity`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `INFO` | No |
+
+Logger severity. Possible values are "INFO", "ERROR", "DEBUG" or "WARN".
```

Comment:
As above, I think we should format the possible values in code style so it's more explicit that these are possible values.